### DISCUSSION TITLE
My react app is blank on gh-pages

### DISCUSSION THREAD

**Discussion by JerryFireman** [community/community#23774] [Other Feature Feedback, Questions, & Ideas]
<p>I uploaded an app I built with Create-React-App to gh-pagesat <a href="https://jerryfireman.github.io/Contacts/" rel="nofollow noopener">https://jerryfireman.github.io/Contacts/</a>.  I have done this previously and it worked fine. But this time all I get is a blank page and the following error messages in the console. </p>
<p><a href="http://5jerryfireman.github.io/:1" class="inline-onebox-loading" rel="nofollow noopener">5jerryfireman.github.io/:1</a> GET <a href="https://jerryfireman.github.io/contacts/static/css/main.b26730bb.chunk.css" rel="nofollow noopener">https://jerryfireman.github.io/contacts/static/css/main.b26730bb.chunk.css</a> net::ERR_ABORTED 404 (Not Found)<br>
manifest.json:1 GET <a href="https://jerryfireman.github.io/contacts/manifest.json" rel="nofollow noopener">https://jerryfireman.github.io/contacts/manifest.json</a> 404 (Not Found)<br>
77Manifest: Line: 1, column: 1, Unexpected token.<br>
76manifest.json:1 GET <a href="https://jerryfireman.github.io/contacts/manifest.json" rel="nofollow noopener">https://jerryfireman.github.io/contacts/manifest.json</a> 404 (Not Found)</p>
<p>Can anyone tell me what I am doing wrong? Thanks!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Your app is receiving 404 errors because it is requesting files that do not exist. Upon looking at the <code>gh-pages</code> branch on your Contacts repository, it appears that the URLs appear in the format of <code>/contacts/static/js/main.124f9f15.chunk.js</code> (note the lowercase <code>c</code> in <code>contacts</code>). Your repository is named <code>Contacts</code> (note the uppercase first character). In your post, you even use the uppercase C in the URL. When I manually requested the URL <code>[https://jerryfireman.github.io/contacts/static/js/main.124f9f15.chunk.js](https://jerryfireman.github.io/contacts/static/js/main.124f9f15.chunk.js)</code>, I get the 404 error. But when I request the URL <code>[https://jerryfireman.github.io/Contacts/static/js/main.124f9f15.chunk.js](https://jerryfireman.github.io/Contacts/static/js/main.124f9f15.chunk.js)</code>, I get the expected contents of the file.</p>
<p>If you change the URLs in your application to use the proper casing for the <code>Contacts</code> path, things should work again.</p>
<p>I hope that helps!</p>

**Comment by JerryFireman** [comment]
<p>Thanks very much lee-dohm! I updated the url and now it is working. I really appreciate your help!</p>

**Comment by cristianvnica** [comment]
<p>For other people who still have this problem, I solved the issue after upgrading ‘gh-pages’ package to the last version.</p>
<p>Also, check if you have this line in package.json: </p>
<pre><code>"homepage": "*your.gh-page.link.goes.here*/",
</code></pre>

**Comment by ChrisJ79** [comment]
<p>I bet that’s the most common mistake for this particular problem…  Nicely done.</p>

**Comment by Mahmoud-farargy** [comment]
<p>Hey, I have this annoying problem for weeks and I can’t solve it. I get a blank page all the time and when I type npm start in my terminal it gives me a bunch of error. I don’t know what’s wrong? I want to learn Vue, angular and react but this problem is always getting in my way.</p>

**Comment by Ktoxcon** [comment]
<p>For all the people who have this problem, if your repository has capital letters, all you have to do is change the name to lowercase only.</p>

**Reply by ghost** [reply]
Thank you for help me on Feb 13, 2023. That was the solution of my problem, the repository name got one capital letter...
